### ISSUE TITLE
Can you share training code?

### ISSUE THREAD

**Issue by Kongjh** [RFOLD/RhoFold#7]
Read the codes, and find 'model.eval', but can not find 'model.train' and the code for Loss function.
Can you share the training codes? But not only share the '[Pretrained model](https://drive.google.com/file/d/1To2bjbhQLFx1k8hBOW5q1JFq6ut27XEv/view?usp=sharing)' parameters weights.

**Comment by Kongjh** [comment]
How can I following the training code to train the model 'from 0 to 1'
